portlets/src/main/webapp/vue-app/engagement-center/components/challenges/Challenges.vue

```diff
@@ -28,7 +28,9 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
           class="btn btn-primary"
           @click="openChallengeDrawer">
           <v-icon small>fas fa-plus</v-icon>
-          <span class="mx-2 d-none d-lg-inline text-capitalize-first-letter subtitle-1">
```

Comment:
shouldn't we just remove the class **text-capitalize-first-letter** and ask to change the text in Crowdin ?